config/settings.yml

```diff
@@ -5,6 +5,13 @@ CACHE_STORE_NAME: 'argo'
 DATE_FORMAT_STR: '%Y-%m-%d %H:%M:%S.%L'
 REINDEX_ON_FLY: false
 
+# Rails application settings
```

Comment:
I'm a bit -1 about having these application configurations here. I think we should use the Rails default configurations per environment checked. Not a show stopper, but this feels a bit off to me.